aldryn_newsblog/admin.py

```diff
@@ -45,6 +45,7 @@ class ArticleAdminForm(TranslatableModelForm):
 
     class Meta:
         model = models.Article
+        fields = '__all__'
```

Comment:
http://stackoverflow.com/questions/28306288/removedindjango18warning-creating-a-modelform-without-either-the-fields-attri